deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -1093,16 +1093,34 @@ def callback_detect_integrity_event(line):
     return None
 
 
-def callback_detect_registry_integrity_state_event(line):
+def callback_detect_registry_integrity_event(line):
+    event = callback_detect_integrity_event(line)
+    if event and event['component'] == 'fim_registry_key':
+        return event
+    if event and event['component'] == 'fim_registry_value':
```

Comment:
The condition `if event` is used so if event is null, the callback is ignored and does not check for the component. If the event was None, and it would try to check the component, it will break because the component key does not exist.